docs/migration/migrate_8_0.asciidoc

```diff
@@ -58,7 +58,8 @@ for example, `logstash-*`.
 ==== Responses are never logged by default
 *Details:* Previously responses would be logged if either `logging.json` was true, `logging.dest` was specified, or a `TTY` was detected.
 
-*Impact:* To restore the previous behavior, in kibana.yml set `logging.events.response=*`.
+*Impact:* To restore the previous behavior, in kibana.yml enable `debug` logs for the `http.server.Kibana.response` context under `logging.loggers`.
+See https://github.com/elastic/kibana/pull/87939 for more details.
```

Comment:
Perhaps we can add an example config in addition to the link to the PR? That way, folks who aren't necessarily interested in the details can enable logs easily.